agent/selfupdater/updater_docker.go

```diff
@@ -112,6 +112,21 @@ func (d *dockerUpdater) CompleteUpdate() error {
 			}
 		}
 
+		// Append /etc/resolv.conf to mount if old container
+		// version is less than v0..0 since /etc/resolv.conf
```

Comment:
the comment mentions v0 but in fact you should mention v0.7.3 here.

Reply:
Good catch, sorry about that. should be fixed.